### ISSUE TITLE
[ISSUE for v2]: Windows does not recognize the input and output correctly

### ISSUE THREAD

**Issue by krilljoy** [w-okada/voice-changer#1591]
### Voice Changer Version

2.0.78-beta cuda

### Operational System

Windows 11

### GPU

Nvidia RTX 5070ti

### CUDA Version

13

### Read carefully and check the options

- [ ] If you use win_cuda_torch_cuda edition, setup cuda? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [x] If you use win_cuda edition, setup cuda and cudnn? [see here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html#requirements)
- [ ] If you use mac edition, client is not launched automatically. Use chrome to open application.?
- [x] I've tried to change the Chunk Size
- [x] I've tried to set the Index to zero
- [x] I've read the [tutorial](https://github.com/w-okada/voice-changer/blob/master/tutorials/tutorial_rvc_en_latest.md)
- [x] I've tried to extract to another folder (or re-extract) the .zip file

### Does pre-installed model work?

No

### Model Type

RVC

### Issue Description

When configuring my system playback and recording devices, Windows gets the virtual cable in the output section, and the output in the input section. 

This means that using any program that doesn't let you set your input device manually cannot utilize w-Okada.

I've combed over the help files but it's mostly all Greek to me - anyone have any insights?  

Thanks in advance!

### Logs on console

torch\nn\utils\weight_norm.py:143: FutureWarning: `torch.nn.utils.weight_norm` is deprecated in favor of `torch.nn.utils.parametrizations.weight_norm`.
  WeightNorm.apply(module, name, dim)
2025-11-17 11:05:09.5024681 [E:onnxruntime:Default, cudnn_fe_call.cc:33 onnxruntime::CudaErrString] No valid engine configs for ConvFwd_
2025-11-17 11:05:09.5082044 [E:onnxruntime:, sequential_executor.cc:572 onnxruntime::ExecuteKernel] Non-zero status code returned while running Conv node. Name:'Conv_18' Status Message: Failed to initialize CUDNN FrontendD:\a\_work\1\s\onnxruntime\core\providers\cuda\cudnn_fe_call.cc:99 onnxruntime::CudaCall D:\a\_work\1\s\onnxruntime\core\providers\cuda\cudnn_fe_call.cc:91 onnxruntime::CudaCall CUDNN_FE failure 8: HEURISTIC_QUERY_FAILED ; GPU=0 ; hostname=PHOENIX ; file=D:\a\_work\1\s\onnxruntime\core\providers\cuda\nn\conv.cc ; line=225 ; expr=s_.cudnn_fe_graph->create_execution_plans({heur_mode});

with the cudnn frontend json:
{"context":{"compute_data_type":"FLOAT","intermediate_data_type":"FLOAT","io_data_type":"FLOAT","name":"","sm_count":-1},"cudnn_backend_version":"9.7.1","cudnn_frontend_version":10700,"json_version":"1.0","nodes":[{"compute_data_type":"FLOAT","dilation":[1,1],"inputs":{"W":"w","X":"x"},"math_mode":"CROSS_CORRELATION","name":"","outputs":{"Y":"::Y"},"post_padding":[32,0],"pre_padding":[31,0],"stride":[1,1],"tag":"CONV_FPROP"}],"tensors":{"::Y":{"data_type":"FLOAT","dim":[158,128,128,1],"is_pass_by_value":false,"is_virtual":false,"name":"::Y","pass_by_value":null,"reordering_type":"NONE","stride":[16384,128,1,1],"uid":3,"uid_assigned":true},"w":{"data_type":"FLOAT","dim":[128,1024,64,1],"is_pass_by_value":false,"is_virtual":false,"name":"w","pass_by_value":null,"reordering_type":"NONE","stride":[65536,64,1,1],"uid":2,"uid_assigned":true},"x":{"data_type":"FLOAT","dim":[158,1024,128,1],"is_pass_by_value":false,"is_virtual":false,"name":"x","pass_by_value":null,"reordering_type":"NONE","stride":[131072,128,1,1],"uid":1,"uid_assigned":true}}}
EP Error: [ONNXRuntimeError] : 11 : EP_FAIL : Non-zero status code returned while running Conv node. Name:'Conv_18' Status Message: Failed to initialize CUDNN FrontendD:\a\_work\1\s\onnxruntime\core\providers\cuda\cudnn_fe_call.cc:99 onnxruntime::CudaCall D:\a\_work\1\s\onnxruntime\core\providers\cuda\cudnn_fe_call.cc:91 onnxruntime::CudaCall CUDNN_FE failure 8: HEURISTIC_QUERY_FAILED ; GPU=0 ; hostname=PHOENIX ; file=D:\a\_work\1\s\onnxruntime\core\providers\cuda\nn\conv.cc ; line=225 ; expr=s_.cudnn_fe_graph->create_execution_plans({heur_mode});

with the cudnn frontend json:
{"context":{"compute_data_type":"FLOAT","intermediate_data_type":"FLOAT","io_data_type":"FLOAT","name":"","sm_count":-1},"cudnn_backend_version":"9.7.1","cudnn_frontend_version":10700,"json_version":"1.0","nodes":[{"compute_data_type":"FLOAT","dilation":[1,1],"inputs":{"W":"w","X":"x"},"math_mode":"CROSS_CORRELATION","name":"","outputs":{"Y":"::Y"},"post_padding":[32,0],"pre_padding":[31,0],"stride":[1,1],"tag":"CONV_FPROP"}],"tensors":{"::Y":{"data_type":"FLOAT","dim":[158,128,128,1],"is_pass_by_value":false,"is_virtual":false,"name":"::Y","pass_by_value":null,"reordering_type":"NONE","stride":[16384,128,1,1],"uid":3,"uid_assigned":true},"w":{"data_type":"FLOAT","dim":[128,1024,64,1],"is_pass_by_value":false,"is_virtual":false,"name":"w","pass_by_value":null,"reordering_type":"NONE","stride":[65536,64,1,1],"uid":2,"uid_assigned":true},"x":{"data_type":"FLOAT","dim":[158,1024,128,1],"is_pass_by_value":false,"is_virtual":false,"name":"x","pass_by_value":null,"reordering_type":"NONE","stride":[131072,128,1,1],"uid":1,"uid_assigned":true}}} using ['CUDAExecutionProvider', 'CPUExecutionProvider']
Falling back to ['CPUExecutionProvider'] and retrying.

**Comment by krilljoy** [comment]
It occurs to me this might be a problem with the virtual cable, and not w-Okada

**Comment by Kuuko-fokkusugaru** [comment]
The virtual cable seems to work fine. I'm not sure what's your issue.
